src/httpd_jsonapi.c

```diff
@@ -1654,6 +1655,17 @@ queue_item_to_json(struct db_queue_item *queue_item, char shuffle)
   safe_json_add_string(item, "composer", queue_item->composer);
   safe_json_add_string(item, "genre", queue_item->genre);
 
+  /* these are represented as strings as json's spec for handling large ints is
+   * messy resulting in values like 
+   *   4811135685787355384 *potentially* becoming 
+   *   4811135685787355000
+   * https://github.com/stedolan/jq/issues/143
+   */
+  sprintf(tmp64,"%" PRIi64, queue_item->songartistid);
```

Comment:
Please use snprintf and check the return value before adding the id to the object.
And please also add a check for songartistid == 0 (this could be the case if it is a queue item that is not part of the library, e. g. spotify tracks)